frontend/src/main/scala/bloop/exec/MultiplexedStreams.scala

```diff
@@ -2,44 +2,29 @@ package bloop.exec
 
 import bloop.logging.{Logger, ProcessLogger}
 
-import java.io.{OutputStream, PrintStream}
-import java.util.UUID
-import java.util.concurrent.{ConcurrentHashMap, ExecutionException, Executors}
+import java.util.concurrent.{ExecutionException, Executors}
+
+import com.martiansoftware.nailgun.ThreadLocalPrintStream
 
 /**
  * Helper object to redirect bytes written to `System.out` and `System.err` to a given logger.
  */
 object MultiplexedStreams {
 
-  /** A combination (stdout, stderr) */
-  private case class Streams(out: PrintStream, err: PrintStream)
+  _initialize()
```

Comment:
Can we not do a more sensible thing than side-effecting in the constructor? Is it possible to have this done explicitly somewhere?